docs/diagrams/ModelClassDiagram.puml

```diff
@@ -33,7 +33,9 @@ Model .right.> ReadOnlyUserPrefs
 Model .left.> ReadOnlyAddressBook
 ModelManager -left-> "1" AddressBook
 ModelManager -right-> "1" UserPrefs
+ModelManager -right-> "1" PredefinedAssignmentData
 UserPrefs .up.|> ReadOnlyUserPrefs
+PredefinedAssignmentData .up |> ReadOnlyPredefinedAssignmentsData
```

Comment:
I think there should be a dot between `|>` and `ReadOnlyPredefinedAssignentData`